activities/Moon.activity/data.ini

```diff
@@ -1,4 +1,5 @@
 [*]
+MoonImageSaved=Image was was saved to journal!
```

Comment:
2 times 'was'. make it one :)

Reply:
I will do that :) in a minute please review